lib/utils.js

```diff
@@ -571,15 +571,30 @@ function deleteLogProperty (log, property) {
 }
 
 /**
- * Filter a log object by removing any ignored keys.
+ * Filter a log object by removing or including keys accordingly.
+ * When `includeKeys` is passed, `ignoredKeys` will be ignored.
+ * One of ignoreKeys or includeKeys must be pass in.
  *
  * @param {object} log The log object to be modified.
- * @param {Set<string> | Array<string>} ignoreKeys An array of strings identifying the properties to be removed.
+ * @param {Set<string> | Array<string> | undefined} ignoreKeys An array of strings identifying the properties to be removed.
+ * @param {Set<string> | Array<string> | undefined} includeKeys An array of strings identifying the properties to be included.
  *
- * @returns {object} A new `log` object instance that does not include the ignored keys.
+ * @returns {object} A new `log` object instance that
+ *  either only includes the keys in ignoreKeys
+ *  or does not include those in ignoredKeys.
  */
-function filterLog (log, ignoreKeys) {
+function filterLog ({ log, ignoreKeys, includeKeys }) {
```

Comment:
Ugh, GitHub won't let me provide a suggestion here due to "deleted lines". We need to tweak the docs to match the new signature:

```js
/**
 * Stuff
 *
 * @param {object} input
 * @param {object} input.log The log object to be modified.
 * @param {...} [input.ignoreKeys] 
 * etc...
 */

Reply:
Ah so sorry i missed that!